core/src/test/scala/integration/kafka/api/BaseAdminIntegrationTest.scala

```diff
@@ -183,12 +184,13 @@ abstract class BaseAdminIntegrationTest extends IntegrationTestHarness with Logg
 
     //with includeAuthorizedOperations flag
     topicResult = getTopicMetadata(client, topic, new DescribeTopicsOptions().includeAuthorizedOperations(true))
-    expectedOperations = AclEntry.supportedOperations(ResourceType.TOPIC).asJava
+    expectedOperations = AclEntry.supportedOperations(ResourceType.TOPIC)
     assertEquals(expectedOperations, topicResult.authorizedOperations)
   }
 
+  @nowarn("cat=deprecation")
   def configuredClusterPermissions: Set[AclOperation] =
-    AclEntry.supportedOperations(ResourceType.CLUSTER)
+    JavaConverters.asScalaSet(AclEntry.supportedOperations(ResourceType.CLUSTER)).toSet
```

Comment:
Any reason why we are using `JavaConverters.asScalaSet` which is deprecated instead of `asScala`.

Reply:
AFAIK `asScala` not exists in scala-2.12 which we use to build kafka.

Reply:
You can use CollectionConverters.

Reply:
Hello @ijuma 

Sorry, I didn't find a way to use `CollectionConverters` to convert java set to immutable scala set.
Can you give me an example? 

`AclEntry.supportedOperations(ResourceType.CLUSTER).asScala` which uses `CollectionConverters` returns `scala.collection.mutable.Set` while here we use `scala.collection.immutable.Set`.

Reply:
You can then call `toSet`. But `CollectionConverters` means you don't have to use deprecated APIs.